src/main.ts

```diff
@@ -15,6 +16,12 @@ export async function runMain<T extends ArgsDef = ArgsDef>(
 ) {
   const rawArgs = opts.rawArgs || process.argv.slice(2);
   const showUsage = opts.showUsage || _showUsage;
+
+  if (!cmd.disableCompletions) {
```

Comment:
i don't think this option is needed, users would opt in completions when they define one.